modules/swagger-servlet/src/test/java/io/swagger/servlet/extensions/ApiOperationTypeTest.java

```diff
@@ -0,0 +1,20 @@
+package io.swagger.servlet.extensions;
+
+import io.swagger.models.Swagger;
+import io.swagger.servlet.Reader;
+import io.swagger.servlet.resources.ApiOperationResource;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+
+public class ApiOperationTypeTest {
+
+    @Test
+    public void testApiOperationInType() {
+
+        final Swagger swagger = new Swagger();
+        Reader.read(swagger, Collections.<Class<?>>singleton(ApiOperationResource.class));
+        Assert.assertNotNull(swagger.getPaths().get("/users"));
+    }
+}
```

Comment:
test should be part of swagger-jaxrs, not swagger-servlet (or even better have them both)

also where does the path "users" come from?

Reply:
It has been extracted from the:

```
@ApiOperation(
        value = "Meta Uber Annotation",
        notes = "Test description",
        consumes = "multipart/form-data",
        produces = "multipart/form-data",
        httpMethod = "GET",
        *nickname = "users",*
```

Reply:
Test added to JaxRS module